tests/src/rules/order.js

```diff
@@ -4228,6 +4228,172 @@ context('TypeScript', function () {
               },
             ],
           }),
+          // Ensure consolidateIslands: 'inside-groups', newlines-between: 'never', and newlines-between-types: 'always-and-inside-groups' do not fight for dominance
```

Comment:
These two tests also cover the uncovered line codecov reported from the previous commit